lib/api/status.js

```diff
@@ -1,49 +1,50 @@
 'use strict';
 
-function configure (app, wares, env) {
+function configure (app, wares) {
   var express = require('express'),
-    api = express.Router( )
-    ;
+      api = express.Router( )
+  ;
 
   api.use(wares.extensions([
     'json', 'svg', 'csv', 'txt', 'png', 'html', 'js'
   ]));
   // Status badge/text/json
-  api.get('/status', function (req, res) {
-    var info = { status: 'ok'
-      , name: app.get('name')
-      , version: app.get('version')
-      , serverTime: new Date().toISOString()
-      , apiEnabled: app.enabled('api')
-      , careportalEnabled: app.enabled('api') && env.settings.enable.indexOf('careportal') > -1
-      , head: wares.get_head( )
-      , settings: env.settings
-      , extendedSettings: app.extendedClientSettings
-    };
-    var badge = 'http://img.shields.io/badge/Nightscout-OK-green';
-    return res.format({
-      html: function ( ) {
-        res.send('<h1>STATUS OK</h1>');
-      },
-      png: function ( ) {
-        res.redirect(302, badge + '.png');
-      },
-      svg: function ( ) {
-        res.redirect(302, badge + '.svg');
-      },
-      js: function ( ) {
-        var head = 'this.serverSettings =';
-        var body = JSON.stringify(info);
-        var tail = ';';
-        res.send([head, body, tail].join(' '));
-      },
-      text: function ( ) {
-        res.send('STATUS OK');
-      },
-      json: function ( ) {
-        res.json(info);
-      }
-    });
+  api.get('/status', function (req, res, next) {
+      var info = { status: 'ok'
+          , apiEnabled: app.enabled('api')
+          , careportalEnabled: app.enabled('api') && app.enabled('careportal')
+          , enabledOptions: app.enabledOptions
+          , defaults: app.defaults
+          , units: app.get('units')
+          , head: wares.get_head( )
+          , version: app.get('version')
+          , thresholds: app.thresholds
+          , alarm_types: app.alarm_types
+          , name: app.get('name')};
+      var badge = 'http://img.shields.io/badge/Nightscout-OK-green';
+      return res.format({
+        html: function ( ) {
+          res.send("<h1>STATUS OK</h1>");
+        },
+        png: function ( ) {
+          res.redirect(302, badge + '.png');
+        },
+        svg: function ( ) {
+          res.redirect(302, badge + '.svg');
+        },
+        js: function ( ) {
+          var head = "this.serverSettings =";
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1052564534/issues/source?bid=2452374&fileBranchId=2883753#l36)